### PR TITLE
Building and deploying

### PR DESCRIPTION
**Pull request by ashioyajotham**

### CODE REVIEW

File: course-02/exercises/udacity-c2-basic-server/src/server.ts [hunk 1]
```diff
@@ -17,7 +17,7 @@ import { Car, cars as cars_list } from './cars';
 
   // Root URI call
   app.get( "/", ( req: Request, res: Response ) => {
-    res.status(200).send("Welcome to the Cloud!");
+    res.status(200).send("Welcome to the Coud!");
```
Comment (Inventrohyder) [inline course-02/exercises/udacity-c2-basic-server/src/server.ts:20]: This should remain as Cloud

File: course-02/exercises/udacity-c2-basic-server/src/server.ts [hunk 2]
```diff
@@ -70,17 +70,78 @@ import { Car, cars as cars_list } from './cars';
...
+
+    // return the resulting list along with 200 success
+    res.status(200).send(cars_list);
+
+  } );
+  
   // @TODO Add an endpoint to get a specific car
   // it should require id
   // it should fail gracefully if no matching car is found
+  app.get("/cars/:id", (req: Request, res: Response) => {
```
Comment (Inventrohyder) [inline course-02/exercises/udacity-c2-basic-server/src/server.ts:92]: Most of the changes in this file were meant as an exercise, therefore, if you need to commit, do so in your personal repo.

You can fork this repo or you can clone and push to your own.

File: course-02/exercises/udacity-c2-restapi/src/config/config.ts [hunk 3]
```diff
@@ -1,22 +1,22 @@
...
   "dev": {
-    "username": "",
-    "password": "",
-    "database": "",
-    "host": "",
+    "username": "postgres",
+    "password": "savagexxiv24",
+    "database": "postgres",
+    "host": "database-1.csavico7ulnc.us-east-1.rds.amazonaws.com",
     "dialect": "postgres",
```
Comment (Inventrohyder) [inline course-02/exercises/udacity-c2-restapi/src/config/config.ts:7]: These changes are mostly specific to your deployment on the cloud.

The blanks are helpful to other students to know that they have to fill them out for their own deployments

File: course-02/exercises/udacity-c2-restapi/src/controllers/v0/users/routes/auth.router.ts [hunk 4]
```diff
@@ -110,11 +110,10 @@ router.post('/', async (req: Request, res: Response) => {
 
     const password_hash = await generatePassword(plainTextPassword);
 
-    const newUser = await new User({
-        email: email,
-        password_hash: password_hash
+    const newUser = await User.build({
```
Comment (Inventrohyder) [inline course-02/exercises/udacity-c2-restapi/src/controllers/v0/users/routes/auth.router.ts:113]: Is there a specific reason to use the `.build` syntax?


Or is it from the package upgrades?

**Review by Inventrohyder — CHANGES_REQUESTED** [review]

**Comment by UAnjali** [comment]
Hi,
This PR contains your personal work that cannot be added to Udacity's repository (starter-code). 
Although, feel free to suggest an update to the starter code. 
Happy Learning
Udacity

**Pull request closed without merging**